tests/PHPUnit/Framework/Mock/FakeAccess.php

```diff
@@ -22,28 +23,36 @@ class FakeAccess extends Access
 {
     public static $superUser = false;
     public static $idSitesAdmin = array();
+    public static $idSitesWrite = array();
     public static $idSitesView = array();
+    public static $idSitesCapabilities = array();
     public static $identity = 'superUserLogin';
     public static $superUserLogin = 'superUserLogin';
 
-    public static function clearAccess($superUser = false, $idSitesAdmin = array(), $idSitesView = array(), $identity = 'superUserLogin')
+    public static function clearAccess($superUser = false, $idSitesAdmin = array(), $idSitesView = array(), $identity = 'superUserLogin', $idSitesWrite = array(), $idSitesCapabilities = array())
     {
         self::$superUser    = $superUser;
         self::$idSitesAdmin = $idSitesAdmin;
+        self::$idSitesWrite = $idSitesWrite;
         self::$idSitesView  = $idSitesView;
         self::$identity     = $identity;
+        self::$idSitesCapabilities = $idSitesCapabilities;
     }
 
     public function getTokenAuth()
     {
         return false;
     }
 
-    public function __construct($superUser = false, $idSitesAdmin = array(), $idSitesView = array(), $identity = 'superUserLogin')
+    public function __construct($superUser = false, $idSitesAdmin = array(), $idSitesView = array(), $identity = 'superUserLogin', $idSitesWrite = array())
     {
-        parent::__construct();
+        // couldn't use DI here as tests seem to fail cause at this time when it is called eg in
+        // plugins/Live/tests/System/ApiCounterTest.php the environment is not set up yet
+        $role = new Access\RolesProvider();
+        $capabilities = new Access\CapabilitiesProvider();
+        parent::__construct($role, $capabilities);
```

Comment:
Think this is because it's used in `provideContainerConfigBeforeClass()`. Might work to set it as `\DI\object(FakeAccess::class)`, but I suppose devs would expect to be able to create a `new FakeAccess()` w/o much trouble.